README.md

```diff
@@ -54,11 +55,16 @@ positional arguments:
 
 optional arguments:
   -h, --help            show this help message and exit
+  --skip                Directory (ie. env/foo) or files (ie. cool.py) to skip
+                        Arguments are seperated by comma (ie. env,kung.py,foo.py)
+  --isort               Only process with isort (imports only)
+  --black               Only process with black (code only)
   --check               don't write changes, just print the files that would
                         be formatted
   --line-length LINE_LENGTH
                         max characters per line; defaults to $MAX_LINE_LENGTH
                         or 100
+  --show-title          show a fancy title at the beginning of pyfmt
```

Comment:
while neat, i don't think anyone would desire this to be on and i'd rather not add superfluous code to the codebase

anyone have an opinion? i'm a fan of less code in general...

Reply:
it's also using bash specific escape codes and not all users are using a bash (or derivative)

Reply:
yeah, I'd rather not have this in here

Reply:
yeah i have no problem with removing it, ... oh well, there was an attempt haha